### PR TITLE
ci: add setup.py to checksum for tox cache

### PR DESCRIPTION
**Pull request by majorgreys**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -81,7 +81,7 @@ commands:
           # In the cache key:
           #   - .Environment.CIRCLE_JOB: We do separate tox environments by job name, so caching and restoring is
           #                              much faster.
-          key: tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}
+          key: tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}-{{ checksum "riotfile.py" }}-{{ checksum "setup.py" }}
```
Comment (brettlangdon) [inline .circleci/config.yml:84]: do we need riotfile here?

File: .circleci/config.yml [hunk 2]
```diff
@@ -90,7 +90,7 @@ commands:
           # In the cache key:
           #   - .Environment.CIRCLE_JOB: We do separate tox environments by job name, so caching and restoring is
           #                              much faster.
-          key: tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}
+          key: tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}-{{ checksum "setup.py" }}
```
Comment (brettlangdon) [inline .circleci/config.yml:93]: do we not need riotfile.py here too?

File: .circleci/config.yml (hunk 1 above)
Comment (majorgreys) [reply .circleci/config.yml:84]: we don't cache riot venvs

Comment (jd) [reply .circleci/config.yml:84]: so you should remove it right? 😁

Comment (brettlangdon) [reply .circleci/config.yml:84]: sorry, let me clarify, why do we have `checksum "riotfile.py"` as part of this cache key?

File: .circleci/config.yml (hunk 2 above)
Comment (brettlangdon) [reply .circleci/config.yml:93]: let me clarify, why do we have `checksum riotfile.py` above, but not here?

File: .circleci/config.yml (hunk 1 above)
Comment (majorgreys) [reply .circleci/config.yml:84]: sorry, pushed this by accident.

**Review by brettlangdon — APPROVED** [review]

**Review by Kyle-Verhoog — APPROVED** [review]

**Pull request merged**